extensions/amp-consent/0.1/consent-info.js

```diff
@@ -191,36 +212,55 @@ export function isConsentInfoStoredValueSame(infoA, infoB, opt_isDirty) {
     } else {
       isDirtyEqual = !!infoA['isDirty'] === !!infoB['isDirty'];
     }
-    return stateEqual && stringEqual && isDirtyEqual;
+    const metadataEqual = isMetadataEqual(
```

Comment:
I introduced this check to help prevent some unnecessary call to the storageAPI. 
Now with the stored value getting more complicated, I am no longer sure if the check itself is worth it...

Reply:
I think it's still worth it, since checking the metadata shouldn't be too much work compared to fetching and parsing info from local storage. Happy to remove the check if you think otherwise.

Reply:
Talked offline: we will measure performance of this later and determine if it should be removed or not.